include/glow/Importer/ProtobufLoader.h

```diff
@@ -256,6 +262,19 @@ class ProtobufLoader {
                                bool isStatic = false, bool isTrainable = false,
                                const std::string &layout = ANY_LAYOUT);
 
+  /// \return whether input/output name \p name of an operator requested in
+  /// FP16 present in \ref operatorPrecisionUpdateInputOutputNames_
+  bool isOpRequestedInFP16Precision(llvm::StringRef name) const;
+
+  /// If \p outputName is specified to run in FP16 precision and output type
+  /// is fp16 add convert to fp32 node and update \ref nodeValueByName_ map,
+  /// so that next operator in the model receives appropriate nodeValue.
+  void updateNodeOutputPrecision(llvm::StringRef outputName, NodeValue NV);
```

Comment:
nit: Since this helper also sets the `nodeValueByName_` mapping and does not do anything to the node output precision at least in the current common case, I'd prefer this function named something like `setNodeOutputMapping()` or something. Doxygen comment can still mention it does the FP16 conversion stuff, but should also be updated to note that this mapping is set.

Reply:
Done.